systemd/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py37
+envlist =
+    py{27,37}-systemd
+    flake8
+
+[testenv]
+usedevelop = true
+platform = linux
```

Comment:
Unit tests work on Mac right?

Reply:
No they don't, only on Linux. As `systemd` is Linux specific, MacOS does not have it and therefore lacks the basic requirements the `pystemd` library relies on.